Source/carthage/Build.swift

```diff
@@ -31,6 +31,12 @@ public struct BuildCommand: CommandType {
 
 				let (stdoutSignal, schemeSignals) = self.buildProjectInDirectoryURL(directoryURL, options: options)
 				stdoutSignal.observe { data in
+					if options.verbose {
+						if let outString = NSString(data: data, encoding: NSUTF8StringEncoding) {
+							carthage.print(outString)
```

Comment:
I like this idea. Having a single destination is cleaner, and I can't think of a situation where I'd need both the inline log *and* the temporary log file.